ci/run-ci.py

```diff
@@ -213,7 +231,7 @@ def get_kube_context():
 def create_cluster_async():
     print('\n# Create GKE cluster')
     cmd = f'gcloud container clusters create {CLUSTER_NAME} '\
-          f'--cluster-version {CLUSTER_VERSION} '\
+          f'--cluster-version {cluster_version()} '\
```

Comment:
Maybe we could simply not specify the `--cluster-version` parameter and let GKE use its default version? I just tested it and it works.

Reply:
Yes. Nevertheless, the default one for GKE is stable, which is 1.16.x. It's different from the bump you've done here : https://github.com/SubstraFoundation/substra-tests/pull/152

Reply:
I think it's ok if we stay on stable. Actually it's probably better to use stable, what do you think?